.github/workflows/prebuilt_binaries.yml

```
@@ -323,13 +323,38 @@ jobs:
             exit 1
           fi
 
-      - name: MPI validation
+      - name: Additional validation (MPI and uninstall)
         uses: ./.github/actions/run-in-docker
         with:
           image: validation:local
           user: cudaq
           shell: bash -l
           run: |
+            # Uninstall CUDA Quantum
+            sudo bash ${CUDA_QUANTUM_PATH}/uninstall.sh -y
+            llvm_dir=`bash -c 'source build_environment.sh && echo $LLVM_INSTALL_PREFIX'`
+            cuquantum_dir=`bash -c 'source build_environment.sh && echo $CUQUANTUM_INSTALL_PREFIX'`
+            cutensor_dir=`bash -c 'source build_environment.sh && echo $CUTENSOR_INSTALL_PREFIX'`
+            if [ -d "$llvm_dir" ] || [ -d "$llvm_dir" ] || [ -d "$llvm_dir" ] ||  [ -d "$CUDA_QUANTUM_PATH" ]; then
+              echo "::error::Some CUDA Quantum artifacts still exist after uninstalling."
+              exit 100
+            fi
+            env -i bash -l
+            if [ -n "$CUDA_QUANTUM_PATH" ] || [ -n "$(echo $PATH | grep -o cudaq)" ]; then
+              echo "::error::Some environment variables still list CUDA Quantum after uninstalling."
+              exit 101
+            fi
+
+            # Re-install CUDA Quantum
+            MPI_PATH=/usr/local/openmpi \
+            sudo -E bash install_cuda_quantum.* --accept > install_log.txt
+            cat install_log.txt
```

Potential simplification?
```suggestion
            sudo -E bash install_cuda_quantum.* --accept | tee install_log.txt
```